src/agent/README.md

```diff
@@ -113,3 +113,43 @@ agent control tool, rather than the Kata runtime.
 
 For further details, see the
 [agent control tool documentation](../tools/agent-ctl/README.md#run-the-tool-and-the-agent-in-the-same-environment).
+
+## Agent options
+
+The kata agent has the ability to configure agent options in guest kernel command line at runtime. Currently, the following agent options can be configured: 
+
+| Option | Name | Description  | Type | Default |
+|-|-|:-:|:-:|:-:|
+|`agent.debug_console` | Debug console flag | Allow to connect guest OS running inside hypervisor Connect using `kata-runtime exec <sandbox-id>` | boolean | `false` |
+| `agent.devmode` | Developer mode | Allow the agent process to coredump | boolean | `false` |
+| `agent.trace` | Trace mode | Allow to static tracing | boolean | `false` |
+| `agent.log` | Log level | Allow the agent log level to be changed (produces more or less output) | string | `"info"` |
+| `agent.server_addr` | Server address | Allow the ttRPC server address to be specified | string | `"vsock://-1:1024"` |
+| `agent.passfd_listener_port` | File descriptor passthrough IO listener port | Allow to set the file descriptor passthrough IO listener port | integer | `0` |
+| `agent.hotplug_timeout` | Hotplug timeout | Allow to configure hotplug timeout of block devices | integer | `3` |
+| `agent.debug_console_vport` | Debug console port | Allow to specify the `vsock` port to connect the debugging console | integer | `0` |
+| `agent.log_vport` | Log port | Allow to specify the `vsock` port to read logs | integer | `0` |
+| `agent.container_pipe_size` | `Container pipe sizes` | Allow to configure the stdout or stderr pipe sizes | integer | `0` |
+| `agent.unified_cgroup_hierarchy` | `Cgroup hierarchy` | Allow to setup v2 cgroups | boolean | `false` |
+| `agent.config_file` | `Configuration file` | Allow to configure options through a configuration file from the root filesystem | string | `""` |
```

Comment:
Optional nit:
I understand that you've done this in the order they appear in config.rs, but I wondered if it might be an idea to sort them alphabetically, so a particular option is quicker to find?

Reply:
Sure, I will do.

Reply:
Fixed.